astro.config.mjs

```diff
@@ -1,5 +1,9 @@
 // @ts-check
-import { defineConfig } from 'astro/config';
+import { defineConfig } from "astro/config";
 
 // https://astro.build/config
-export default defineConfig({});
+export default defineConfig({
+  redirects: {
+    "/prototypes/draper/": "/prototypes/draper/index.html",
```

Comment:
Will `/prototypes/draper/` also apply to `/prototype/draper`? 🤔

Reply:
We'd have to create a new redirect for it. For simplicity, this might be the best route.

Otherwise, we can look at [rewrites](https://docs.astro.build/en/guides/routing/#rewrites).

Reply:
Verified via testing that the ending `/` didn't seem to make a difference either way. So leaving it as is.